qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -94,15 +96,9 @@ def _choose_layout_condition(property_set):
         return not property_set["layout"]
 
     def _trivial_not_perfect(property_set):
-        # Verify that a trivial layout is perfect. If trivial_layout_score > 0
-        # the layout is not perfect. The layout is unconditionally set by trivial
-        # layout so we need to clear it before contuing.
-        if (
-            property_set["trivial_layout_score"] is not None
-            and property_set["trivial_layout_score"] != 0
-        ):
-            return True
-        return False
+        # Verify that a trivial layout is perfect.  If perfect, then the circuit should already
+        # appear to be swap mapped.
+        return property_set["is_swap_mapped"] is not None and not property_set["is_swap_mapped"]
```

Comment:
Pre-existing, but the above comment still seems inconsistent. This seems to be checking both that `CheckMap` ran and that it didn't find the trivial layout to be perfect.

Reply:
Done in f1a0bbab2.